### PR TITLE
feat: Add Netlify readme

### PR DESCRIPTION
**Pull request by colomolo**
Local development setup

### CODE REVIEW

File: apps/netlify/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Run lambda with command:
+```
+npm run serve
+```
+
+This script expects the following environment variables to be set:
+
+| Variable                      | Description                                                               |
+| ---                           | ---                                                                       |
+| `APP_IDENTITY_PRIVATE_KEY`    | Private key of the key pair generated via App definition page |
```
Comment (kdamball) [inline apps/netlify/README.md:42]: Maybe explain how to copy this correctly, i.e: download the private key file from security tab, copy/paste the content of the file to a string as an env variable. I find this to be a tricky part for most people to get

File: apps/netlify/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+Netlify App
+===
+
+## Local Development
+
+In order to develop Netlify App locally you will need to run locally both the Netlify Contentful
```
Comment (andipaetzold) [inline apps/netlify/README.md:6]:
```suggestion
In order to develop Netlify App locally, you will need to run both the Netlify Contentful 
```

File: apps/netlify/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+### Launching the lambda
+First, start HTTP tunnel with ngrok:
+```
+ngrok http 8000
+```
+
+Enable events and add https ngrok url to the events config on App definition page as follows:
+`https://<ngrok url>/app-events`
+
+Go to app's Security tab. Generate key pair and signing secret. These are needed as environment variables (see below). For example you can copy private key contents and paste it as a value to `APP_IDENTITY_PRIVATE_KEY` variable.
```
Comment (andipaetzold) [inline apps/netlify/README.md:30]:
```suggestion
Go to the app's Security tab. Generate a key pair and a signing secret. These are needed as environment variables (see below). For example, you can copy the private key and paste it as a value to the `APP_IDENTITY_PRIVATE_KEY` variable.
```

File: apps/netlify/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+You would also want to enable Entry and Asset publish/unpublish events:
```
Comment (andipaetzold) [inline apps/netlify/README.md:32]:
```suggestion
To receive Entry and Asset publish/unpublish events, configure the Events tab the following:
```

**Review by andipaetzold — APPROVED** [review]
Just some grammar/wording

**Review by andipaetzold — APPROVED** [review]

**Pull request merged**